rules/DowngradePhp80/Rector/FuncCall/DowngradeStrContainsRector.php

```diff
@@ -89,12 +89,12 @@ public function refactor(Node $node): Identical | NotIdentical | null
     private function matchStrContainsOrNotStrContains(Expr $expr): ?FuncCall
     {
         $expr = ($expr instanceof BooleanNot) ? $expr->expr : $expr;
-
-        if ($expr instanceof FuncCall) {
-            if ($this->isName($expr, 'str_contains')) {
-                return $expr;
-            }
+        if (! $expr instanceof FuncCall) {
+            return null;
+        }
+        if (! $this->isName($expr, 'str_contains')) {
+            return null;
         }
-        return null;
+        return $expr;
```

Comment:
This will be fixed in `main` soon :)

Reply:
This is done by rectify automatically.

Reply:
Yes, now its fixed. Rebase should remove it